Dockerfile.const

```diff
@@ -11,6 +11,7 @@ RUN if [ -e /certs/cert.pem ]; then cp /certs/cert.pem /etc/pki/tls/certs/ca-bun
 # fetch scripts/guardian-set-init.sh deps
 RUN dnf -y install jq
 RUN dnf -y install make
+RUN dnf -y install findutils
```

Comment:
Acting as the ghost of @SEJeff past, he would suggest we combine these three lines into a single install call to reduce the size of the Dockerfile images.. maybe for another PR when we finally speed up Dockerfile.const